src/pudl/package_data/eia923/column_maps/cooling_system_information.csv

```diff
@@ -6,26 +6,26 @@ cooling_id_eia,cooling_system_id,cooling_system_id,cooling_system_id,cooling_sys
+monthly_maximum_discharge_temperature_f,,,discharge_maximum_temperature,discharge_maximum_temperature,discharge_maximum_temperature_f,discharge_maximum_temperature_f,discharge_maximum_temperature_f,discharge_maximum_temperature_f,discharge_maximum_temperature_f,discharge_maximum_temperature_f,discharge_maximum_temperature_f,discharge_maximum_temperature_f,discharge_maximum_temperature_f,discharge_maximum_temperature_f,discharge_maximum_temperature_f
 temperature_method,,,method_for_temperatures,method_for_temperatures,method_for_temperatures,method_for_temperatures,method_for_temperatures,method_for_temperatures,method_for_temperatures,method_for_temperatures,method_for_temperatures,method_for_temperatures,method_for_temperatures,method_for_temperatures,method_for_temperatures
-diversion_volume_million_gallons,,,,,diversion_volume_million_gallons,diversion_volume_million_gallons,diversion_volume_million_gallons,diversion_volume_million_gallons,diversion_volume_million_gallons,diversion_volume_million_gallons,diversion_volume_million_gallons,diversion_volume_million_gallons,diversion_volume_million_gallons,diversion_volume_million_gallons,diversion_volume_million_gallons
-withdrawal_volume_million_gallons,,,,,withdrawal_volume_million_gallons,withdrawal_volume_million_gallons,withdrawal_volume_million_gallons,withdrawal_volume_million_gallons,withdrawal_volume_million_gallons,withdrawal_volume_million_gallons,withdrawal_volume_million_gallons,withdrawal_volume_million_gallons,withdrawal_volume_million_gallons,withdrawal_volume_million_gallons,withdrawal_volume_million_gallons
-discharge_volume_million_gallons,,,,,discharge_volume_million_gallons,discharge_volume_million_gallons,discharge_volume_million_gallons,discharge_volume_million_gallons,discharge_volume_million_gallons,discharge_volume_million_gallons,discharge_volume_million_gallons,discharge_volume_million_gallons,discharge_volume_million_gallons,discharge_volume_million_gallons,discharge_volume_million_gallons
-consumption_volume_million_gallons,,,,,consumption_volume_million_gallons,consumption_volume_million_gallons,consumption_volume_million_gallons,consumption_volume_million_gallons,consumption_volume_million_gallons,consumption_volume_million_gallons,consumption_volume_million_gallons,consumption_volume_million_gallons,consumption_volume_million_gallons,consumption_volume_million_gallons,consumption_volume_million_gallons
-chlorine_annual_1000_lbs,annual_chlorine,annual_chlorine,,,,,,,,,,,,,
-annual_withdrawal_rate_cubic_feet_per_second,annual_withdrawal_rate,annual_withdrawal_rate,,,,,,,,,,,,,
```

Comment:
PUDL standard is to minimize abbreviations unless absolutely ridiculous (e.g., multi-letter acronyms for leak detection protocols that are widely known), so these should actually stay `cubic_feet_per_second` and `fahrenheit` as Austen originally had them.